src/Microsoft.AspNetCore.SignalR/DefaultHubLifetimeManager.cs

```diff
@@ -106,6 +106,9 @@ public override Task InvokeUserAsync(string userId, string methodName, object[]
 
         public override Task OnConnectedAsync(HubConnectionContext connection)
         {
+            // Set the hub groups feature
+            connection.Features.Set<IHubGroupsFeature>(new HubGroupsFeature());
```

Comment:
I confused this OnConnectedAsync with the one that is on Hub so ignore my comment.